src/Components/Inputs/Select/UpSelect.tsx

```diff
@@ -198,6 +200,12 @@ export default class UpSelect extends BaseControlComponent<UpSelectProps, any> {
         newState.extra.menuIsOpen = false;
       }
 
+      if (newState == null && this.state.extra !== null) {
+        const extra = this.state.extra === undefined || this.state.extra === null ? {} : { ...this.state.extra };
```

Comment:
if (newState == null && this.state.extra != null) { // null & undefined
        const extra = { ...this.state.extra };